<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clientswindow</class>
 <widget class="QWidget" name="clientswindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>671</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>480</y>
     <width>141</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Скрыть клиентов</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>480</y>
     <width>201</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Найти клиента по паспорту</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>510</y>
     <width>221</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Найти клиента по ФИО</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>470</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>450</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ФИО</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>500</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Паспорт</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>520</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>450</y>
     <width>221</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Посмотреть всех клиентов</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
